app/views/users/show.html.erb

```diff
@@ -1,28 +1,33 @@
 <div>
-  <section>
-    <div>User Image</div>
-    <div>
-      <h1><a><%= @user.name %></a></h1>
-      <span>Number of posts:<%= @user.posts_counter %></span>
+  <section class="users-ul">
+    <div class="users-li">
+    <div><%= image_tag @user.photo %></div>
+    <div class="user-info">
+      <h2><a><%= @user.name %></a></h2>
+      <p>Number of posts:<%= @user.posts_counter %></p>
+    </div>
     </div>
   </section>
 
-  <section>
+  <section class="user-data">
     <h2>Bio</h2>
     <p><%= @user.bio %></p>
   </section>
 
   <section>
-   <ul>
+   <ul class="users-ul">
      <% @user.posts.each do |post| %>
-       <li>
+       <li class="user-data">
          <h3>Post <%='#'%><%= post.id %></h3>
-         <p>post.text</p>
+         <p><%= post.text %></p>
+        <div class="lk-cm">
          <span>Comments: <%= post.comments_counter %></span>
          <span>Likes: <%= post.likes_counter %></span>
+        </div>
        </li>
      <% end %>
    </ul>
-   <button>See all posts</button>
```

Comment:
- [ ] Kindly make sure you use the `#last_three_posts` method to render the user's 3 most recent posts. your implementation should stick as much as possible to this format [format given](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png) for the project requirements and make sure you match its **[design page](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)**.